packages/react-charting/src/components/AreaChart/AreaChart.test.tsx

```diff
@@ -190,3 +191,21 @@ describe('Render calling with respective to props', () => {
     renderMock.mockRestore();
   });
 });
+
+describe('AreaChart - mouse events', () => {
+  beforeEach(sharedBeforeEach);
+  afterEach(sharedAfterEach);
+
+  it('Should render callout on hover', () => {
+    const root = document.createElement('div');
+    document.body.appendChild(root);
+
+    // document.getElementbyId() returns null if component is not attached to DOM
+    wrapper = mount(<AreaChart data={chartPoints} calloutProps={{ doNotLayer: true }} />, { attachTo: root });
+    wrapper.find('rect').simulate('mouseover');
+    const tree = toJson(wrapper, { mode: 'deep' });
+    expect(tree).toMatchSnapshot();
```

Comment:
Also test callout for stack and custom callouts

Reply:
Try moving the mouse hover and observe the callout changing positions and values.

Reply:
Okay